upgrade/1.5.2/README.md

```diff
@@ -0,0 +1,307 @@
+# CSM 1.5.2 Patch Installation Instructions
+
+* [Introduction](#introduction)
+* [Bug fixes and improvements](#bug-fixes-and-improvements)
+* [Steps](#steps)
+
+## Introduction
+
+This document guides an administrator through the patch update to Cray Systems Management `v1.5.2`
+from CSM `v1.5.0`. If upgrading from CSM `v1.4.x`,  then follow the procedures
+described in [CSM major/minor version upgrade](../README.md#csm-majorminor-version-upgrade) instead.
+In the unusual situation of upgrading from a pre-release version of CSM `v1.5.0`, then follow the procedures
+described in [CSM major/minor version upgrade](../README.md#csm-majorminor-version-upgrade) instead.
+
+If there are more recent CSM `v1.5` patch versions available, note that there is no need to perform
+intermediate CSM `v1.5` patch upgrades. Instead, consider upgrading to the latest CSM `v1.5`
+patch release. See [CSM patch version upgrade](../README.md#csm-patch-version-upgrade) for the full
+list of patch versions.
+
+## Bug fixes and improvements
+
+INSTRUCTIONS FOR CASM FOLKS (delete this line later): Replace any line below that contains your
+name with a one line description of the change that occurred as a result of your Jira. Combine
+bullets together if it makes sense (eg. many Paradise bullets were combined into a single "Support
+for Paradise hardware" bullet..
+
+* Added support for Paradise hardware
+* Fixed issue in `conman` in order to support Paradise hardware
+* Fixed issue in `run_hms_ct_tests.sh` which caused false positives for CDU switches
+* Enhanced `hmcollector` logging
+* Mike Buchmann: CASMHMS-6207
+* Mitchell Harding: CASMCMS-8971
+* Mitchell Harding: CASMCMS-8973
+* Mitchell Harding: CASMCMS-8978
+* Mitchell Harding: CASMCMS-8186
+* Mitchell Harding: CASMCMS-8438
+* Mitchell Harding: CASMINST-6861
+* Mitchell Harding: CASMTRIAGE-6993
+* Mitchell Harding: CASMTRIAGE-7019
+* Mitchell Harding: CASMHMS-5838
+* Mitchell Harding: CASMCMS-9001
```

Comment:
This is a change to fix a problem with our builds, but not a change that is reflected in the product itself.
```suggestion
```